### PR TITLE
Upgrade Chinese translation

### PR DESCRIPTION
**Pull request by Augustakit**
Corrected some Chinese mistakes to make it look more natural

### CODE REVIEW

**Comment by jokoho48** [comment]
hey, thanks for the contribution but can you please fix the merge conflicts?

**Comment by Augustakit** [comment]
OK, I just corrected it. Please check if there are any errors

File: addons/wp/stringtable.xml [hunk 1]
```diff
@@ -170,7 +170,7 @@
             <Polish>Zadanie - Atak/Odwrót</Polish>
             <Russian>Задача Нападение/Отступление</Russian>
             <Italian>Compito Assalto/Ritirata</Italian>
-            <Chinesesimp>撤退/突袭任务</Chinesesimp>
+            <Chinesesimp突袭/撤退任务</Chinesesimp>
```
Comment (jokoho48) [inline addons/wp/stringtable.xml:173]: > missing here
```suggestion
            <Chinesesimp>突袭/撤退任务</Chinesesimp>
```

Comment (Augustakit) [reply addons/wp/stringtable.xml:173]: Sorry, I accidentally deleted something when copying files. Thank you for your correction

**Pull request merged**